models/comments_models.js

```diff
@@ -11,4 +12,16 @@ function fetchCommentsByArticleId(article_id){
     })
 }
 
-module.exports = { fetchCommentsByArticleId }
\ No newline at end of file
+function updateCommentByArticleId(article_id, commentObject){
+    const {username, body} = commentObject
+    if(body === ""){
+        return Promise.reject({status: 400, msg: "Bad request-The comment is empty"})
+    }
+    return db.query(`INSERT into comments (body, article_id,author) VALUES ($1,$2,$3) RETURNING *`, [body,article_id,username])
+    .then(({rows}) => {
+        const {author, body} = rows[0]
+        const comment = {author, body}
+        return comment;
+    })
```

Comment:
^^ this is where you will make changes because of the test :) Have a look at what is in rows[0]! :)